client/src/models/Organization.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react'
 
 import Model from 'components/Model'
+import utils from 'utils'
 import Settings from 'Settings'
 
+import * as yup from 'yup'
+
+export const advisorOrganization = Settings.fields.advisor.org
```

Comment:
The reason to define a const in the model is to avoid having the same Setting.… in a thousand places, which makes it very hard to reorganise the dictionary should we want to. Hence the move of commonly used settings to the corresponding model class. I agree the name is not ideal.

Reply:
I think I understand a bit better why that change. I am still thinking that:
- if we start making copies of the references to nested settings it may make it really hard to make the settings reloadable on the fly. The point of the Settings is to provice a single end point to query settings during the rendering, making local copies will makes this difficult.
- already later in client/src/models/Person.js there is a reference to `advisorPerson.name` so that is misleading
- We start to have 2 diverging hierarchies `Settings.fields.principal.person.name` vs `PersonDefs.principalPerson.name`. Either way, we should pick one canonical hierarchy and stick to it.
- having to rely on the module user to do `import * as PersonDefs from 'models/Person'` is a bit strange. It is not obcious that `import *` is getting defs. Also then we have a Person class attached to PersonDefs. Would have been better to do `import {Defs} from 'models/Person'`. 
- I suggest to move all the `fieldLabels` to the dictionary and use that instead. If `Settings.` needs to be made better, let's do that. Is there a reason not to add them to the dictionary?